frontend/src/components/worksheets/items/SchemaItem/SchemaItem.js

```diff
@@ -116,7 +117,9 @@ class SchemaItem extends React.Component<{
         const schemaHeaders = schemaItem.header;
         let newRow = { from_schema_name: schemaItem.schema_name };
         schemaHeaders.forEach((header) => {
-            newRow[header] = '';
+            // Set post-processor to null for new rows so we can tell if a row has been autofilled.
+            // This also makes new row default values consistent with the first row.
```

Comment:
What does this mean?

Reply:
We need to set the default post-processor to null so we can tell the difference between initialized value and when user manually deleted the input for post-processor.

Reply:
> We need to set the default post-processor to null so we can tell the difference between initialized value and when user manually deleted the input for post-processor.

I think that's what the first comment ("Set post-processor to null for new, unedited rows to indicate that the row can be autofilled.") means. I don't understand what the second comment ("This also makes new row default values consistent with the first row.") means. Can you modify it to make it clearer?

Reply:
> 
> 
> > We need to set the default post-processor to null so we can tell the difference between initialized value and when user manually deleted the input for post-processor.
> 
> I think that's what the first comment ("Set post-processor to null for new, unedited rows to indicate that the row can be autofilled.") means. I don't understand what the second comment ("This also makes new row default values consistent with the first row.") means. Can you modify it to make it clearer?

The old code sometime sets post-processor to `null` and sometimes `''` and it's confusing.
The second line was referring to this issue. But I this the first line of the comment should be enough to explain why `null` is used so I removed the second line.